test/mocks/helpers/almaRefundHelpers.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+
+var proxyquire = require('proxyquire')
+    .noCallThru()
+    .noPreserveCache();
+
+var transaction = {
+    wrap: function (callBack) {
+        return callBack.call();
+    }
+};
+
+var alma = {
+    refundPayment: function () {
+        return function () {
+            return {
+                msg: 'OK',
+                call: function () {
```

Comment:
there's a sonarcloud code smell about this line, do you know how to see them ? if you don't let's check it together ;).